src/components/common/applicant-info-2.vue

```diff
@@ -238,7 +238,7 @@ export default class ApplicantInfo2 extends NameRequestMixin {
   ]
   phoneRules = [
     v => !!v || 'Required field',
-    v => (v.length <= 30) || 'Cannot exceed 30 characters'
+    v => (!v || v.length <= 30) || 'Cannot exceed 30 characters'
```

Comment:
When v is null/undefined (which is weird because the first rule should have caught this, but it has definitely happened according to Sentry), this causes an error, which I think might be trashing the other rule validations, and letting required fields be empty.

So this may the fix for /bcgov/entity#6470.